flink-table/flink-table-api-java/src/main/java/org/apache/flink/table/operations/ShowDatabasesOperation.java

```diff
@@ -21,8 +21,37 @@
 /** Operation to describe a SHOW DATABASES statement. */
 public class ShowDatabasesOperation implements ShowOperation {
 
+    private final String catalogName;
+    private final boolean notLike;
+    private final String likePattern;
+
+    public ShowDatabasesOperation() {
+        this(null, false, null);
+    }
+
+    public ShowDatabasesOperation(String catalogName, boolean notLike, String likePattern) {
+        this.catalogName = catalogName;
+        this.notLike = notLike;
+        this.likePattern = likePattern;
+    }
+
+    public String getCatalogName() {
+        return catalogName;
+    }
+
+    public boolean isNotLike() {
+        return notLike;
+    }
+
+    public String getLikePattern() {
+        return likePattern;
+    }
+
     @Override
     public String asSummaryString() {
+        if (likePattern != null) {
```

Comment:
You can refer to other implementations to see if it need a  preposition, from, in or other keywords here.

Reply:
OK